cmd/urfave-cli-genflags/main.go

```diff
@@ -313,6 +313,10 @@ func (ft *FlagType) GenerateVisibleFlagInterface() bool {
 	return ft.skipInterfaceNamed("VisibleFlag")
 }
 
+func (ft *FlagType) GeneratePersistentFlagInterface() bool {
+	return ft.skipInterfaceNamed("PersistentFlag")
```

Comment:
I'm realizing now that this function is confusingly named 😅 TODO for later! renaming so that it reads `return ft.needsGeneratedInterfaceNamed(...)`

Reply:
Yes it is !!!!!